examples/complete/main.tf

```diff
@@ -62,3 +66,96 @@ module "rds_instance" {
 
   context = module.this.context
 }
+
+# The remainder of this configuration has to do with creating an S3 integration.
+# This is to test var.role_associations.
+# See: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance_role_association
+# and https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/oracle-s3-integration.html
+locals {
```

Comment:
can we keep locals at the top?

Reply:
Keeping locals at the top is definitely the style we usually want, but in this case I created at a certain point near the bottom of the file because it is part of a section that is only applicable to the `testExamplesS3Integration` test case.